### PR TITLE
V bpastiu/issue181

### PR DESCRIPTION
**Pull request by v-bpastiu**

### CODE REVIEW

File: 08-kubernetes-sealed-secrets/README.md [hunk 1]
```diff
@@ -103,19 +103,19 @@ The output looks similar to:
 
 ```text
 NAME                            CHART VERSION   APP VERSION     DESCRIPTION
-sealed-secrets/sealed-secrets   2.1.6          v0.17.5         Helm chart for the sealed-secrets controller.
+sealed-secrets/sealed-secrets   2.4.0           v0.18.1         Helm chart for the sealed-secrets controller.
```
Comment (v-ctiutiu) [inline 08-kubernetes-sealed-secrets/README.md:106]: After running:

```console
helm repo add sealed-secrets https://bitnami-labs.github.io/sealed-secrets
helm repo update sealed-secrets
helm search repo sealed-secrets
```

I get:

```text
NAME                         	CHART VERSION	APP VERSION	DESCRIPTION                                       
bakito/sealed-secrets-web    	2.8.0        	v2.8.0     	A web interface for Sealed Secrets by Bitnami.    
bitnami/sealed-secrets       	1.0.10       	0.18.1     	Sealed Secrets are "one-way" encrypted K8s Secr...
sealed-secrets/sealed-secrets	2.5.2        	v0.18.1    	Helm chart for the sealed-secrets controller.
```

I see that **v2.5.2** of the **sealed-secrets/sealed-secrets** Helm chart is mapped to the **v0.18.1** of the app. Can you double check please?

File: 09-scaling-application-workloads/assets/manifests/prometheus-adapter-values-v3.3.1.yaml [hunk 2]
```diff
@@ -14,12 +14,12 @@ prometheus:
 replicas: 2
 
 # See: https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/docs/config.md
```
Comment (v-ctiutiu) [inline 09-scaling-application-workloads/assets/manifests/prometheus-adapter-values-v3.3.1.yaml:16]: Shouldn't this section be commented until the reader gets to the respective section in the guide?

**Review by v-ctiutiu — CHANGES_REQUESTED** [review]

File: 08-kubernetes-sealed-secrets/README.md (hunk 1 above)
Comment (v-bpastiu) [reply 08-kubernetes-sealed-secrets/README.md:106]: Double checked. After I upgrade to 2.4.0 and do an `helm ls -n sealed-secrets` I get this:
Screenshot 2022-08-03 at 14 15 45
Apparently the app version has not changed in a while.

File: 09-scaling-application-workloads/assets/manifests/prometheus-adapter-values-v3.3.1.yaml (hunk 2 above)
Comment (v-bpastiu) [reply 09-scaling-application-workloads/assets/manifests/prometheus-adapter-values-v3.3.1.yaml:16]: Indeed. My bad. I left it uncommented.

**Review by v-ctiutiu — APPROVED** [review]
Looks good to me.

**Pull request merged**